code/core/src/main/java/com/adobe/marketing/mobile/internal/eventhub/history/AndroidEventHistory.java

```diff
@@ -83,8 +83,8 @@ public void run() {
      * @param handler {@code EventHistoryResultHandler<Integer>} containing the the total number of
      *     matching events in the {@code EventHistoryDatabase} if an "any" search was done. If an
      *     "ordered" search was done, the handler will contain a "1" if the event history requests
-     *     were found in the order specified in the eventHistoryRequests array and a "0" if the
-     *     events were not found in the order specified.
+     *     were found in the order specified in the eventHistoryRequests array, a "0" if the events
+     *     were not found in the order specified, and a "-1" if the database failure occurred.
```

Comment:
We return -1 regardless if it is ordered/any search. Can you fix this comment to reflect the same?